services/oauth/oauth.go

```diff
@@ -33,6 +33,7 @@ type (
 
 const (
 	OAuth           AuthType = "OAuth"
+	OAuthAccounts   AuthType = "OAuth-Account"
```

Comment:
OAuthAccount

Reply:
In db-config of zoho_dev we are calling it `OAuth-Account`. Should I change there as well?